app/src/main/java/cm/aptoide/pt/search/view/SearchResultFragment.java

```diff
@@ -931,12 +932,20 @@ private void setupToolbar() {
     }
   }
 
-  public void enableUpNavigation() {
-    if (actionBar != null) {
-      actionBar.setHomeButtonEnabled(true);
-      actionBar.setDisplayHomeAsUpEnabled(true);
-      actionBar.setDisplayShowHomeEnabled(true);
-    }
+  public void removeBakcButton() {
+    handler.post(new Runnable() {
+      @Override
+      public void run() {
+        if (toolbar != null) {
+          for (int i=0; i<toolbar.getChildCount(); i++) {
+            View v = toolbar.getChildAt(i);
+            if (toolbar.getChildAt(i) instanceof  androidx.appcompat.widget.AppCompatImageButton) {
```

Comment:
Yes. Back button is only one AppCompatImageButton on this toolbar and no, we're not able to get it by id, because it is added dynamically by searchview item. 
Standarad ways of disabling this back button doesn't work in this case.

Reply:
OK - I figured you'd looked into it, but I just wanted to be sure.

Reply:
Also fixed if condition.